wallet/res/navigation/nav_home.xml

```diff
@@ -40,6 +40,19 @@
         <action
             android:id="@+id/home_to_contacts"
             app:destination="@id/contactsFragment" />
+
+        <action
+            android:id="@+id/home_to_notifications"
+            app:destination="@id/showNotificationsFragment">
+
+            <argument
+                android:name="ShowNavBar"
+                android:defaultValue="false" />
```

Comment:
This argument is `false` by default.

Reply:
I see this code repeating many times in nav_home.xml

Reply:
Oh, those other places are from other code I wrote before regarding Masternode Keys.